pokemonterminal/terminal/adapters/iterm.py

```diff
@@ -13,10 +13,10 @@ class ItermProvider(_TProv):
     end tell"""
 
     def is_compatible() -> bool:
-        return os.environ.get("ITERM_PROFILE")
+        return "ITERM_PROFILE" in os.environ
 
     def __run_osascript(stream):
-        p = subprocess.Popen(['osascript'], stdout=subprocess.PIPE,
+        p = subprocess.Popen(["osascript"], stdout=subprocess.PIPE,
```

Comment:
One style thing, that I don't feel super strongly about but is probably worth mentioning based on how you changed a lot of quoting in one of your commits, in python it's pretty standard to use single quotes for internal strings (ie strings that don't get printed to console or to file etc), and double quotes for external strings

Reply:
I wasn't aware this was a convention. I just noticed the difference in quotation style and decided to make quotations consistent.